src/fxa-geodb.js

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@shane-tomlinson I was simply copying the structure we have for fxa-easter-egg and fxa-password-strength-checker, should this be changed to `lib`? If so, then where do I put the `errors.js` and `defaults.js` files?